packages/design-tokens/package.json

```diff
@@ -21,14 +21,14 @@
   "license": "MIT",
   "sideEffects": false,
   "scripts": {
-    "test": "cd ../../ && yarn jest ./packages/design-tokens",
-    "build": "yarn clean && yarn prepublishOnly",
-    "build:json": "yarn ts-node --files ./bin/buildCssVarTokens.ts",
+    "test": "cd ../../ && pnpm jest ./packages/design-tokens",
```

Comment:
Weird, why is this here? Does it not get run through the root command?

Reply:
Lol I don't know, it definitely does